packages/web-components/src/components/va-minimal-header/va-crisis-line-modal.scss

```diff
@@ -0,0 +1,171 @@
+$red-60v: #b51d09;
+$red-70v: #8b1303;
+
+$small-desktop-screen: 1008px;
```

Comment:
Is this the right place to be adding tokens? Do we have these values somewhere else in a central place we could pull from instead?

Reply:
This might be irrelevant if we are able to just use the existing crisis modal variation from `va-modal`: https://design.va.gov/storybook/?path=/docs/components-va-modal--crisis-line-modal